### PR TITLE
Polish translation

### PR DESCRIPTION
**Pull request by TexturedPolak**
May be not 100% accurate

### CODE REVIEW

**Comment by TexturedPolak** [comment]
But that's is 100% completed :)

File: lib/l10n/app_pl.arb [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+  "add_count_to_queue": "Dodaj ({count}) do Kolejki",
+  "play_count_next": "Odtwórz ({count}) następne",
+  "album": "Album",
+  "copied_to_clipboard": "Skopiowano {data} do schowka",
+  "add_to_following_playlists": "Dodano {track} do danych Playlist",
+  "add": "Dodaj",
+  "added_track_to_queue": "Dodano {track} do kolejki",
+  "add_to_queue": "Dodano do kolejki",
+  "track_will_play_next": "{track} następny",
+  "play_next": "Odtwórz nastepny",
```
Comment (karniv00l) [inline lib/l10n/app_pl.arb:78]:
```suggestion
  "play_next": "Odtwórz następny",
```

File: lib/l10n/app_pl.arb [hunk 2]
```diff
@@ -0,0 +1,260 @@
...
+  "removed_track_from_queue": "Usunięto {track} z kolejki",
+  "remove_from_queue": "Usunięto z kolejki",
+  "remove_from_favorites": "Usunięto z ulubionych",
+  "save_as_favorite": "Zapisz do ulubionych",
+  "add_to_playlist": "Dodaj do playlisty",
+  "remove_from_playlist": "Usuń z playlisty",
+  "add_to_blacklist": "Dodaj do czarnej listy",
+  "remove_from_blacklist": "Usuń z czarnej listy",
+  "share": "Udostępnij",
+  "mini_player": "Mały otwarzacz",
```
Comment (karniv00l) [inline lib/l10n/app_pl.arb:88]:
```suggestion
  "mini_player": "Mały odtwarzacz",
```

File: lib/l10n/app_pl.arb [hunk 3]
```diff
@@ -0,0 +1,260 @@
...
+  "slide_to_seek": "Przesuń aby przewinąć do przodu lub do tyłu.",
+  "shuffle_playlist": "Odtwarzaj losowo z  playlisty",
+  "unshuffle_playlist": "Nie odtwarzaj losowo z  playlisty",
+  "previous_track": "Poprzedni utwór",
+  "next_track": "Następny utwór",
+  "pause_playback": "Zatrzymaj otwarzanie",
```
Comment (karniv00l) [inline lib/l10n/app_pl.arb:94]:
```suggestion
  "pause_playback": "Zatrzymaj odtwarzanie",
```

File: lib/l10n/app_pl.arb [hunk 4]
```diff
@@ -0,0 +1,260 @@
...
+  "resume_playback": "Wznów otwarzanie",
```
Comment (karniv00l) [inline lib/l10n/app_pl.arb:95]:
```suggestion
  "resume_playback": "Wznów odtwarzanie",
```

File: lib/l10n/app_pl.arb [hunk 5]
```diff
@@ -0,0 +1,260 @@
...
+  "version": "Wersja",
+  "build_number": "Numer Build'a",
+  "founder": "Założyciel",
+  "repository": "Repozytorium",
+  "bug_issues": "Błędy",
+  "made_with": "Stworzono z ❤️ w Bangladesh'u 🇧🇩",
+  "kingkor_roy_tirtho": "Kingkor Roy Tirtho",
+  "copyright": "© 2021-{current_year} Kingkor Roy Tirtho",
+  "license": "Licencja",
+  "add_spotify_credentials": "Dodaj swoje dane logowania Spotify aby zacząć",
```
Comment (karniv00l) [inline lib/l10n/app_pl.arb:161]:
```suggestion
  "add_spotify_credentials": "Dodaj swoje dane logowania Spotify, aby zacząć",
```

File: lib/l10n/app_pl.arb [hunk 6]
```diff
@@ -0,0 +1,260 @@
...
+  "credentials_will_not_be_shared_disclaimer": "Nie martw się, żadne dane logowania nie są zbierane ani udostępniane nikomu",
+  "know_how_to_login": "Nie wiesz jak się zalogować?",
```
Comment (karniv00l) [inline lib/l10n/app_pl.arb:163]:
```suggestion
  "know_how_to_login": "Nie wiesz, jak się zalogować?",
```

File: lib/l10n/app_pl.arb [hunk 7]
```diff
@@ -0,0 +1,260 @@
...
+  "follow_step_by_step_guide": "Postępuj zgodnie z poradnikiem krok po kroku",
+  "spotify_cookie": "Spotify {name} Ciasteczek",
+  "cookie_name_cookie": "{name} Ciasteczko",
+  "fill_in_all_fields": "Proszę wypełnić wszystkie pola",
+  "submit": "Zatwierdź",
+  "exit": "Zamknij",
+  "previous": "Poprzedny",
```
Comment (karniv00l) [inline lib/l10n/app_pl.arb:170]:
```suggestion
  "previous": "Poprzedni",
```

File: lib/l10n/app_pl.arb [hunk 8]
```diff
@@ -0,0 +1,260 @@
...
+  "moderate": "Umiarkowany",
+  "deselect_all": "Odznacz wszystkie",
+  "select_all": "Zaznacz wszystkie",
+  "are_you_sure": "Jesteś pewny?",
+  "generating_playlist": "Generowanie twojej własnej playlisty...",
+  "selected_count_tracks": "Wybrano {count} utworów",
+  "download_warning": "Jeśli hurtowo pobierasz wszystkie utwory, wyraźnie piracisz muzykę i wyrządzasz szkody kreatywnej społeczności muzycznej. Mam nadzieję, że jesteś tego świadomy. Zawsze staraj się szanować i wspierać ciężką pracę Artysty",
+  "download_ip_ban_warning": "Przy okazji, Twój adres IP może zostać zablokowany w YouTube z powodu nadmiernych żądań pobierania niż zwykle. Blokada IP oznacza, że nie możesz korzystać z YouTube (nawet jeśli jesteś zalogowany) przez co najmniej 2-3 miesiące z tego urządzenia IP. Spotube nie ponosi żadnej odpowiedzialności, jeśli tak się stanie",
+  "by_clicking_accept_terms": "Klikając 'Akceptuj' zgadzasz się z następującymi warunkami:",
+  "download_agreement_1": "Wiem, że pirace muzykę. Jestem zły.",
```
Comment (karniv00l) [inline lib/l10n/app_pl.arb:229]:
```suggestion
  "download_agreement_1": "Wiem, że piracę muzykę. Jestem zły.",
```

File: lib/l10n/app_pl.arb [hunk 9]
```diff
@@ -0,0 +1,260 @@
...
+  "download_agreement_2": "Będę wpierał artystę i robię to tylko dlatego,że nie mam pieniędzy na albumy wykonawcy. ",
```
Comment (karniv00l) [inline lib/l10n/app_pl.arb:230]:
```suggestion
  "download_agreement_2": "Będę wpierał artystę i robię to tylko dlatego, że nie mam pieniędzy na albumy wykonawcy. ",
```

**Comment by TexturedPolak** [comment]
thanks for fast review :)

File: lib/l10n/app_pl.arb [hunk 10]
```diff
@@ -0,0 +1,260 @@
...
+  "dark": "Ciemny",
+  "light": "Jasny",
+  "system": "Systemowy",
+  "accent_color": "Kolor Akcentu",
+  "sync_album_color": "Synchronizuj kolor albumu",
+  "sync_album_color_description": "Używa dominującego koloru okładki albumu jako koloru akcentującego",
+  "playback": "Odtwarzanie",
+  "audio_quality": "Jakość dźwięku",
+  "high": "Duża",
+  "low": "Mała",
```
Comment (karniv00l) [inline lib/l10n/app_pl.arb:133]: If this is related to audio quality, then it should be:

```suggestion
  "high": "Wysoka",
  "low": "Niska",
```

File: lib/l10n/app_pl.arb [hunk 11]
```diff
@@ -0,0 +1,260 @@
...
+  "hours": "{hours} Godziny",
+  "hour": "{hours} Godzina",
+  "custom_hours": "Własne godziny",
+  "logs": "Logi",
+  "developers": "Developerzy",
+  "not_logged_in": "Nie jesteś zalogowany",
+  "search_mode": "Tryb szukania",
+  "youtube_api_type": "Typ API",
+  "ok": "Ok",
+  "failed_to_encrypt": "Nie można rozszyfrować :(",
```
Comment (karniv00l) [inline lib/l10n/app_pl.arb:255]:
```suggestion
  "failed_to_encrypt": "Nie można zaszyfrować :(",
```

File: lib/l10n/app_pl.arb [hunk 12]
```diff
@@ -0,0 +1,260 @@
...
+  "accept": "Akceptuj",
+  "details": "Szczegóły",
+  "youtube": "YouTube",
+  "channel": "Kanał",
+  "likes": "Polubienia",
+  "dislikes": "Nie lubi",
+  "views": "Wyświetlenia",
+  "streamUrl": "URL strumienia",
+  "stop": "Stop",
+  "sort_newest": "Sortuj po dodanych najpóźniej",
```
Comment (karniv00l) [inline lib/l10n/app_pl.arb:242]:
```suggestion
  "sort_newest": "Sortuj według ostatnio dodanych",
```

File: lib/l10n/app_pl.arb [hunk 13]
```diff
@@ -0,0 +1,260 @@
...
+  "sort_oldest": "Sortuj po dodanych najwcześniej",
```
Comment (karniv00l) [inline lib/l10n/app_pl.arb:243]:
```suggestion
  "sort_oldest": "Sortuj według najstarszych dodanych",
```

File: lib/l10n/app_pl.arb [hunk 14]
```diff
@@ -0,0 +1,260 @@
...
+  "save_as_favorite": "Zapisz do ulubionych",
+  "add_to_playlist": "Dodaj do playlisty",
+  "remove_from_playlist": "Usuń z playlisty",
+  "add_to_blacklist": "Dodaj do czarnej listy",
+  "remove_from_blacklist": "Usuń z czarnej listy",
+  "share": "Udostępnij",
+  "mini_player": "Mały odwarzacz",
+  "slide_to_seek": "Przesuń, aby przewinąć do przodu lub do tyłu.",
+  "shuffle_playlist": "Odtwarzaj losowo z  playlisty",
+  "unshuffle_playlist": "Nie odtwarzaj losowo z  playlisty",
```
Comment (karniv00l) [inline lib/l10n/app_pl.arb:91]:
```suggestion
  "shuffle_playlist": "Odtwarzaj losowo z playlisty",
  "unshuffle_playlist": "Nie odtwarzaj losowo z playlisty",
```

File: lib/l10n/app_pl.arb [hunk 15]
```diff
@@ -0,0 +1,260 @@
...
+  "max": "Maksymalnie",
+  "target": "Cel",
+  "moderate": "Umiarkowanie",
+  "deselect_all": "Odznacz wszystkie",
+  "select_all": "Zaznacz wszystkie",
+  "are_you_sure": "Jesteś pewny?",
+  "generating_playlist": "Generowanie twojej własnej playlisty...",
+  "selected_count_tracks": "Wybrano {count} utworów",
+  "download_warning": "Jeśli hurtowo pobierasz wszystkie utwory, wyraźnie piracisz muzykę i wyrządzasz szkody kreatywnej społeczności muzycznej. Mam nadzieję, że jesteś tego świadomy. Zawsze staraj się szanować i wspierać ciężką pracę Artysty",
+  "download_ip_ban_warning": "Przy okazji, Twój adres IP może zostać zablokowany w YouTube z powodu nadmiernych żądań pobierania niż zwykle. Blokada IP oznacza, że nie możesz korzystać z YouTube (nawet jeśli jesteś zalogowany) przez co najmniej 2-3 miesiące z tego urządzenia IP. Spotube nie ponosi żadnej odpowiedzialności, jeśli tak się stanie",
```
Comment (karniv00l) [inline lib/l10n/app_pl.arb:227]:
```suggestion
  "download_ip_ban_warning": "Przy okazji, Twój adres IP może zostać zablokowany w YouTube z powodu nadmiernych żądań pobierania niż zwykle. Blokada IP oznacza, że nie możesz korzystać z YouTube (nawet jeśli jesteś zalogowany) przez co najmniej 2-3 miesiące z IP tego urządzenia. Spotube nie ponosi żadnej odpowiedzialności, jeśli tak się stanie",
```

File: lib/l10n/app_pl.arb (hunk 10 above)
Comment (TexturedPolak) [reply lib/l10n/app_pl.arb:133]: That's better looks in create playlist ;)

Comment (TexturedPolak) [reply lib/l10n/app_pl.arb:133]: But that's also related to audio quality 🤷‍♂️

Comment (karniv00l) [reply lib/l10n/app_pl.arb:133]: I guess we need another key then

**Comment by KRTirtho** [comment]
You guys are doing great :fire: 

Let me know if you're done. I'm planning a release today (with the fix to the recent YouTube API change)
It'll be nice if we can include Polish translation as well :smile:

**Comment by KRTirtho** [comment]
Oh that's great :smile: 

Thanks for the contribution @TexturedPolak :rocket: 
Thanks for the review @karniv00l :muscle:

**Comment by TexturedPolak** [comment]
Not problem ;)

**Pull request merged**